util/index.ts

```diff
@@ -147,22 +156,23 @@ const getBtcExchangePrice = (fiatAmount, satsAmount) => {
   }
 };
 
-const objectToArray = object => {
-  const array = [];
+const objectToArray = (object: any): any[] => {
+  const array: any[] = [];
 
   for (const i in object) array.push(object[i]);
 
   return array;
 };
 
+
```

Comment:
@Mersho let's get rid of unnecessary whitespace changes

Reply:
Done.